project-base/app/src/DataFixtures/Demo/GoPayDataFixture.php

```diff
@@ -166,6 +166,7 @@ private function getGoPayPaymentMethodData(
         $goPayPaymentMethodData->imageLargeUrl = 'https://gate.gopay.cz/images/checkout/' . $data['image_large_url'] . '.png';
         $goPayPaymentMethodData->paymentGroup = $data['payment_group'];
         $goPayPaymentMethodData->domainId = $domainId;
+        $goPayPaymentMethodData->available = true;
```

Comment:
nitpick, it maybe we want an unavailable payment method in the data fixtures as well? I do not insist on that

Reply:
good idea, added